app/models/concerns/custom_field_addable.rb

```diff
@@ -118,6 +124,12 @@ def custom_field?(field_identifier)
     self.class.custom_field?(field_identifier, model: self)
   end
 
+  def unanswered?(field_identifier)
+    value = custom_value(field_identifier)
+    field = custom_field(field_identifier)
+    value.blank? && field.descendants.all? { |i| custom_value(i.id).blank? }
+  end
```

Comment:
As for your question, though, this method is for determining if the question or any of its descendants are answered for this loan. So something like `value.blank?` didn't seem right, because a `value` shouldn't need to know anything about its field's descendants—a question can be answered and also have children, remember. It could be `field.unanswered_in?(value_set)` or `field.unanswered_for?(loan)`, but are those any better?